exercises/practice/hello-world/.docs/instructions.append.md

```diff
@@ -1,6 +1,6 @@
 # Hints
 
-To complete this exercise, you need to implement the `hello` function.
+To complete this exercise, you need to define the constant `hello`.
 
 You will find the type signature for `hello` already in place,
```

Comment:
type signature is a term only applicable to functions.

```suggestion
You will find the type for `hello` already in place,
```

Reply:
Do you have a source on this? I tentatively disagree.

Reply:
I was going to say https://en.wikipedia.org/wiki/Type_signature, but for Haskell specifically, the report supersedes, so it is correct to call it a signature. But I suspect (without proof) that it may be seen as incorrect in common language. Is "type annotation" better according to either criterion?

Reply:
I understand «type annotation» to mean what the report calls «[expression type-signature](https://www.haskell.org/onlinereport/haskell2010/haskellch3.html#x8-560003.16)». (In Haskell. E.g. in Python it's different.)

I do not know much about common use of «type signature» outside of Haskell. But 'Inside' Haskell it seems unnatural to me to reserve it for declarations of the form `name :: type -> …` and to use a different term for the other such declarations. Also, I cannot remember ever having seen a distinction being made before, hence my surprise.